scripts/change-log-generator-utils.js

```diff
@@ -46,7 +46,7 @@ function getPreviousReleaseBranch(releaseBranch) {
  function getReleaseBranches() {
   return shell
     .exec(
-      `git branch --remotes --list --sort='-creatordate' '${constants.RELEASE_BRANCH_PREFIX}*'`,
+      `git branch --remotes --list --sort='-creatordate' '${constants.REMOTE_RELEASE_BRANCH_PREFIX}*'`,
```

Comment:
Is this what fixes our change log issue?

Reply:
Yes, I think so. But I don't have the ability to test it since now v54.10.0 has been cut and there is no feat/fix PRs coming after it. So, I created this PR as a draft. Once we get PRs in this week, I can test if this is really the root cause of change log issue.